app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectedJenkinsServers.tsx

```diff
@@ -92,6 +92,7 @@ const ConnectedJenkinsServers = ({ connectedJenkinsServer }: ConnectedStateProps
 								</>
 								<div className={cx(connectedStateCell, connectedStateCellEvent)}>
 									{mapLastEventStatus(pipeline.lastEventStatus)} {pipeline.lastEventType}
+									{' '}{pipeline.environmentNames?.join(', ')}
```

Comment:
/nit and potentially a design question but do we want to wrap it in brackets? so it becomes
`Successful deployment (stg, prod, etc)`